src/pot.erl

```diff
@@ -119,21 +119,20 @@ valid_totp(Token, Secret, Opts) ->
 -spec time_interval(proplist()) -> time().
 time_interval(Opts) ->
   IntervalLength = proplists:get_value(interval_length, Opts, 30),
-  AddSeconds = proplists:get_value(addwindow, Opts, 0) * proplists:get_value(interval_length, Opts, 30),
```

Comment:
Could you please explain to me why this behavior should be changed? We extensively use this in our application tests.